stack.yaml

```diff
@@ -52,26 +51,23 @@ extra-deps:
 # Windows terminal.
 # https://github.com/judah/haskeline/pull/126
 - github: judah/haskeline
-  commit: d6c2643b0d5c19be7e440615c6f84d603d4bc648
-- guid-0.1.0@sha256:a7c975be473f6f142d5cc1b39bc807a99043d20b1bb0873fdfe7a3ce84d2faf1,1078
-- sandi-0.5@sha256:b278d072ca717706ea38f9bd646e023f7f2576a778fb43565b434f93638849aa,3010
-- fuzzyfind-3.0.0@sha256:d79a5d3ed194dd436c6b839bf187211d880cf773b2febaca456e5ccf93f5ac65,1814
-- monad-validate-1.2.0.0@sha256:9850f408431098b28806dd464b6825a88a0b56c84f380d7fe0454c1df9d6f881,3505
-- NanoID-3.1.0@sha256:9118ab00e8650b5a56a10c90295d357eb77a8057a598b7e56dfedc9c6d53c77d,1524
-# 2.3.27 bundles sqlite >=3.35.0, needed for 'delete returning'
-- direct-sqlite-2.3.27
-# not in lts-18.13
-- recover-rtti-0.4.0.0@sha256:2ce1e031ec0e34d736fa45f0149bbd55026f614939dc90ffd14a9c5d24093ff4,4423
-- lock-file-0.7.0.0@sha256:3ad84b5e454145e1d928063b56abb96db24a99a21b493989520e58fa0ab37b00,4484
-- http-client-0.7.11
-- lsp-1.5.0.0
-- lsp-types-1.5.0.0
-- text-rope-0.2@sha256:53b9b4cef0b278b9c591cd4ca76543acacf64c9d1bfbc06d0d9a88960446d9a7,2087
+  commit: 3b26f1e6c279e9bb5cd052c34c852aa8f8cfa17e
```

Comment:
Sanity check: did you ensure color renders properly in a Windows terminal?

Reply:
I'm checking.

Reply:
This commit is morally the same: the PR that we were pointing at was rebased by the author atop the latest main so it continues to build. The PR still isn't merged :/

Reply:
<img width="754" alt="image" src="https://user-images.githubusercontent.com/538571/205144712-19b164ff-6f88-48a4-9b20-4ce2e1e6e817.png">
This is what I get; let me check again on trunk.

Reply:
Oh. Sorry for making you do all of that, but I understand now.

I too went looking for "what is commit `d6c2643b0d5c19be7e440615c6f84d603d4bc648`?" when trying to fix compiler errors with `haskeline` due to the bump, and I saw it in the history of the open PR we were depending on:

![image](https://user-images.githubusercontent.com/1074598/205182881-14f9d51b-3aee-4188-8260-12257db830ea.png)

But I got the history wrong. Looks like the author was not just force-pushing over their PR to keep it atop the main branch, they changed the implementation, too. 

![image](https://user-images.githubusercontent.com/1074598/205183021-f47766c5-073e-444f-bd18-5af013513aa5.png)

Meanwhile @ChrisPenner (maybe independently) discovered that the original approach was correct, so pointed us at `d6c2643b0d5c19be7e440615c6f84d603d4bc648`.

So I believe the correct move is to grab `haskeline` master (since it builds), and revert https://github.com/judah/haskeline/commit/f827f10145380ec9e685a72e4b8a7a27ef6717ea on top of it (which is how `d6c2643b0d5c19be7e440615c6f84d603d4bc648` came to be).

Reply:
@aryairani I've updated this dependency. Now it's pointing at our `unisonweb/haskeline` fork, branch `lts-20.0`. The contents of the branch are just `judah/haskeline` HEAD, plus [this commit](https://github.com/unisonweb/haskeline/commit/6df6d5a0711dd1056b6182559281b31ecfc24773), which is equivalent to [the previous commit we were using](https://github.com/judah/haskeline/commit/d6c2643b0d5c19be7e440615c6f84d603d4bc648) (although now rebased on top of `judah/haskeline` HEAD, because the old commit didn't build against `lts-20.0`).

Some confusion: we have a [unison branch](https://github.com/unisonweb/haskeline/tree/unison) (the default branch) in our `unisonweb/haskeline` fork, 7 commits ahead and 13 commits behind `aryairiani/haskeline`. I first tried merging `judah/haskeline` HEAD into this, but got a ton of conflicts. Is this branch just dead code at this point?

Reply:
> Is this branch just dead code at this point?

I would guess so.

Reply:
screenshot of 20dacc9a0dca7f14997973f1d54da3a4796bb2c6  courtesy of @tarikozkanli
![image](https://user-images.githubusercontent.com/538571/209356011-b14c7678-0e96-4f88-a1e2-613f68b63a4f.png)